### PR TITLE
COMMERCE-10358 Re-elaborate functional test CanAssertDiagramNameAndDescriptionOnTheProductCard

### PR DESCRIPTION
**Pull request by baharturk**
https://issues.liferay.com/browse/COMMERCE-10358

### CODE REVIEW

**Comment by baharturk** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONProductsAPI.macro [hunk 1]
```diff
@@ -132,7 +132,7 @@ definition {
 		}
 
 		if (!(isSet(shortDescription))) {
-			var shortDescription = "";
+			var shortDescription = "${shortDescription}";
```
Comment (victoroscuro) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONProductsAPI.macro:135]: @baharturk can you please revert this one? This should remain as it is now on master

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceProducts.macro [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+		CommerceJSONProductsAPI._addCommerceProduct(
+			catalogName = "${catalogName}",
+			productName = "${productName}",
+			productType = "Diagram",
+			shortDescription = "${shortDescription}",
+			sku = "${sku}");
+
+		CommerceProducts.openProductsAdmin();
+
+		CommerceNavigator.searchEntry(entryName = "Diagram");
```
Comment (victoroscuro) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceProducts.macro:13]: @baharturk the search can be done by name directly

**Review by victoroscuro — CHANGES_REQUESTED** [review]

**Comment by baharturk** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONProductsAPI.macro (hunk 1 above)
Comment (victoroscuro) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONProductsAPI.macro]: @baharturk you can use macro CommerceProducts.searchAndGoToProductEntry

**Comment by baharturk** [comment]
ci:test:commerce

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceProducts.macro (hunk 2 above)
Comment (victoroscuro) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceProducts.macro:13]: @baharturk you can use macro CommerceProducts.searchAndGoToProductEntry

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**